monkey/monkey_island/cc/repositories/utils/agent_configuration_schema_compiler.py

```diff
@@ -61,15 +39,25 @@ def _add_plugins(self, schema: Dict[str, Any]) -> Dict[str, Any]:
 
         return schema
 
-    @staticmethod
-    def _get_plugin_type_string(plugin_type: AgentPluginType):
-        return plugin_type.name.lower()
+    def _add_properties_to_objects(self, schema: Dict[str, Any]) -> Dict[str, Any]:
```

Comment:
```suggestion
    def _add_properties_field_to_plugin_types(self, schema: Dict[str, Any]) -> Dict[str, Any]:
```

^^ Not sure about this name, but something more descriptive would be good.